pocean/dsg/trajectoryProfile/cr.py

```diff
@@ -154,7 +154,11 @@ def to_dataframe(self, clean_cols=True, clean_rows=True, **kwargs):
         # integer. Each value in the index variable is the zero-based trajectory
         # index that the profile belongs to i.e. profile p belongs to trajectory
         # i=trajectory_index(p), as in section H.2.5.
-        r_index_var = self.filter_by_attrs(instance_dimension=lambda x: x is not None)[0]
+        r_index_var = self.filter_by_attrs(instance_dimension=lambda x: x is not None)
+        if not r_index_var:
+            raise ValueError('Could not find the `instance_dimension` attribute in {!r}'.format(self))
```

Comment:
I don't really like to output the `repr` here. Do you have a better idea that is more informative than that?